src/seedsigner/models/settings.py

```diff
@@ -173,3 +171,27 @@ def debug(self) -> bool:
         return self._data[SettingsConstants.SETTING__DEBUG] == SettingsConstants.OPTION__ENABLED
 
 
+    def microsd_handler(self, action, mount_dir):
+        from seedsigner.controller import Controller
+        if Controller.HOSTNAME == Controller.SEEDSIGNER_OS:
+        
+            if action == "add":
```

Comment:
Any strings that coordinate actions across different parts of the code would ideally be defined as constants, even for simple cases like "add" and "remove". Maybe in `MicroSD`? It's not really a model but the name would make it dead-obvious: `MicroSD.ACTION__ADD` or something.

Reply:
And does "add" refer to adding persistent settings in this context or is it more like "SD card inserted"? If the latter, then `MicroSD.ACTION__INSERTED` and `.ACTION__REMOVED` would be more descriptive.